catalog/edf-methanesat-ee/projects_edf-methanesat-ee_assets_public-preview_L4point.jsonnet

```diff
@@ -0,0 +1,162 @@
+local id = 'projects/edf-methanesat-ee/assets/public-preview/L4point';
+local subdir = 'edf-methanesat-ee';
+local version = '1.0.0';
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local methane = importstr 'methane.md';
+local publications = importstr 'publications.md';
+local units = import 'units.libsonnet';
+local license = spdx.proprietary;
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+  title: 'MethaneSAT L4 Point Sources ' + version,
+  version: version,
+  description: |||
+    The methane emission fluxes were produced using a point source detection and
+    emissions quantification framework specialized to exploit the high spatial
+    resolution, wide spatial coverage, and high precision of MethaneSAT data
+    (methodology is described in [Chulakdabba et al. (2023)](https://doi.org/10.5194/amt-16-5771-2023).)
+    The point source quantification framework was extensively tested in blind
+    controlled-release experiments as detailed in [Chulakdabba et al. (2023)](https://doi.org/10.5194/amt-16-5771-2023)
+    and [Abbadi et al. (2024)](https://doi.org/10.1021/acs.est.4c02439). Not all
+    data products are available for all collections.
+  ||| + publications,
+  'gee:categories': ['atmosphere'],
+  keywords: [
+    'climate',
+    'emissions',
+    'ghg',
+    'methane',
+    'edf',
+    'methanesat',
+  ],
+  providers: [
+    ee.producer_provider('Environmental Defense Fund - MethaneSAT', 'https://methanesat.org'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  // TODO(supertom): I don't think I should need an end date?
+  extent: ee.extent_global('2024-06-14T00:00:00Z', '2025-01-17T00:00:00Z'),
+  summaries: {
+    platform: ['MethaneSAT'],
+    'gee:schema': [
+      {
+        name: 'plume_id',
+        description: 'Plume id (unique per satellite observation).',
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'flux',
+        description: 'Methane flux quantification.',
+        units: units.kg_per_hour,
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'flux_hi',
```

Comment:
Could you please list them alphabetically and make sure they correspond to the properties in the asset?

I don't see, eg, flux_hi and flux_lo in the properties:
https://code.earthengine.google.com/cda2a9059523faa5e17f31ee649fd874

collection_id: 067906E0
date: 01/17/2025
flux: 1821.667034
flux_sd: 1239.08052
plume_id: 067906E0_0
plume_id_in_scene: 0
region: Uinta
target_id: 110

Reply:
Thanks.  I've added the properties to the COGs and updated the `gee::schema` to match.